### ISSUE TITLE
Distended Jugular Lymphatic Sacs

### ISSUE THREAD

**Issue by EDempseyUK** [obophenotype/human-phenotype-ontology#6992]
**Preferred term label:**

Distended jugular lymphatic sacs
**Synonyms**

**Definition (free text, please give PubMed ID)**
The jugular lymphatic sacs (JLS) are a physiological and temporary part of fetal lymphatic development. They are formed from small buds of lymphatic endothelial cells arising from the internal jugular veins.  By 14 weeks gestation they have developed into lymphatic nodes which drain into the systemic circulation. Visibility of JLS on ultrasound past 14 weeks of gestation implies distension and this may be associated with raised nuchal translucency.

PMID:23718767, 19248150, 20227670

**Parent term (use hpo.jax.org/app)**
HP:0010880

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**

Distended JLS may pre-date or accompany raised nuchal translucency or cystic hygroma and are associated with the same range of disorders. These are largely chromosomal aneuploidies or significant copy number variation. Current evidence suggests distended JLS in the absence of raised NT are likely to be normal variation.

**Your nano-attribution (ORCID)**

0000-0002-7653-4653

**Comment by patt1morgan** [comment]
@sparkste, @Ragla  Thank you for agreeing to work with us to advance the perinatal section of the HPO. We would appreciate it if you could review this new term request and approve it or help us improve it. Any suggestions welcome!

**Comment by pnrobinson** [comment]
Possible parent term
Abnormal sonographic appearance of the fetal neck
-- this term
-- Increased nuchal translucency
-- fetal cystic hygroma
-- (no separate term) nuchal edema (make this a synonym of Increased nuchal translucency)

**Comment by pnrobinson** [comment]
Separate new term.
-- Thick nuchal fold (used in second trimester)
--Increased nuchal fold -- synonyms of increased N

**Comment by pnrobinson** [comment]
Thanks @EDempseyUK and sorry for the delay. I have added the term as a child of a new parent term called Fetal neck anomaly
